### DISCUSSION TITLE
How to Set Default Code Editor Theme???

### DISCUSSION THREAD

**Discussion by ivanwebdevs** [prasathmani/tinyfilemanager#696] [General]
Please Help!!!... how to set default for Code Editor Theme..... I always do select theme when open my code

**Comment by prasathmani** [comment]
![image](https://user-images.githubusercontent.com/11345598/147633877-74415ec2-add9-4ebd-857a-7568baa0d5d3.png)
```js
editor.setTheme("ace/theme/twilight"); //set your default theme
```

**Reply by ivanwebdevs** [reply]
thanks
